examples/sagaMonitor/index.js

```diff
@@ -392,6 +393,10 @@ if(IS_BROWSER) {
   window.$$LogSagas = logSaga
 }
 
+if (IS_REACT_NATIVE) {
```

Comment:
could you refactor it, so both `IS_BROWSER` and `IS_REACT_NATIVE ` cases are handled at the same time? some kind of obtaining global object at the top and checking here if it got obtained